docs/customization/emails.md

```diff
@@ -23,10 +23,10 @@ Source emails use [MJML](https://mjml.io/) and must be compiled to HTML before u
 
 ### Before you start
 
-To compile the emails you need to have node.js installed.
+To compile the emails you need to have Node.js installed.
 Version 10 or later is required. See the [installation instructions](https://nodejs.org/en/download/package-manager/).
```

Comment:
Node JS 8 or greater actually (please check, but AFAIK that's the MJML requirement) which would be easier for non-rolling release distribution users.

Reply:
Node.js 8 is no longer supported: https://github.com/nodejs/Release#release-schedule. I guess it's fine as it is.